spec/lib/roman_numerals/converter_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe RomanNumerals::Converter do
+  subject(:converter) { RomanNumerals::Converter }
+
+  base_digits = {
+    1    => 'I',
+    4    => 'IV',
+    5    => 'V',
+    9    => 'IX',
+    10   => 'X',
+    40   => 'XL',
+    50   => 'L',
+    90   => 'XC',
+    100  => 'C',
+    400  => 'CD',
+    500  => 'D',
+    900  => 'CM',
+    1000 => 'M'
+  }
+
+  describe ".to_decimal" do
```

Comment:
Do we want a to_roman spec or are we not going to use it?

Reply:
Yes we do: we need it when we capture the section having decimal digits.
In the representation, we want to show the Roman form (I guess ... it looks nicer)